### ISSUE TITLE
-L (--list) should run app in nogui mode

### ISSUE THREAD

**Issue by buddhi1980** [buddhi1980/mandelbulber2#28]
By default it tries to setup gui, so it crash in headless terminal. It should switch to nogui mode

**Comment by zebastian** [comment]
it should exit, before starting the gui or coreapplication:
```
void cCommandLineInterface::ReadCLI (void)
{
	...
	// list parameters only
	if(cliData.listParameters)
	{
		...
		out.flush();
		exit(0);
```

**Comment by buddhi1980** [comment]
It was needed to switch to nogui mode before allocation of gApplication. Otherwise it caused following error (when application was started from tty):

```
Log file name: /home/krzysztof/.mandelbulber_log.txt
Detected 8 CPUs
Default data directory: /home/krzysztof/.mandelbulber/
Fatal: QXcbConnection: Could not connect to display 
((null):0, (null))
```
Fixed in commit 3b30d17
